### PR TITLE
Update DeepSemgrep installation instructions

### PR DESCRIPTION
**Pull request by adamkvitek**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs

### CODE REVIEW

**Comment by adamkvitek** [comment]
Preview of the updated section:
Screenshot 2022-12-07 at 18 58 46

File: docs/deepsemgrep.md [hunk 1]
```diff
@@ -1,37 +1,61 @@
...
+
+To install DeepSemgrep, follow these steps:
+
+1. Log in to Semgrep CLI with the following command:
     ```sh
     semgrep login
     ```
-4. Follow the link that Semgrep prints to the command-line.
-5. To install DeepSemgrep, use the following command:
+1. Follow the link that Semgrep printed in the command-line.
```
Comment (s-santillan) [inline docs/deepsemgrep.md:40]:
```suggestion
1. Follow the link that Semgrep printed in the command line.
```

Only hyphenate "command line" when it is used as an adjective. I'm linking MS style guide, but this is a general English rule, so it applies to words such as "hard-won" as well. https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/command-line

File: docs/deepsemgrep.md [hunk 2]
```diff
@@ -1,37 +1,61 @@
...
+1. To install DeepSemgrep, use the following command:
     ```sh
     semgrep install-deep-semgrep
     ```
-6. To test DeepSemgrep, use the following command with your own rules:
-    ```sh
-    semgrep --deep --config=myrule.yaml
+1. To test DeepSemgrep, use the following command:
```
Comment (s-santillan) [inline docs/deepsemgrep.md:45]:
```suggestion
1. To test DeepSemgrep, use the following command in the root directory of the codebase to scan:
```
We don't tell people where (what directory) to run the command in.

**Review by s-santillan — APPROVED** [review]
left 2 comments, lgtm overall

File: docs/deepsemgrep.md (hunk 1 above)
Comment (adamkvitek) [reply docs/deepsemgrep.md:40]: I agree, this must have slipped through my Grammarly check. Thanks! @s-santillan

File: docs/deepsemgrep.md (hunk 2 above)
Comment (adamkvitek) [reply docs/deepsemgrep.md:45]: @s-santillan great idea!

**Review by milanwilliams — APPROVED** [review]

**Pull request merged**